dns/dnssec.py

```diff
@@ -29,7 +29,7 @@
 import dns.rdataclass
 
 class UnsupportedAlgorithm(dns.exception.DNSException):
-    """Raised if an algorithm is not supported."""
+    """The DNSSEC algorithm is not supported."""
     pass
```

Comment:
just a hint: if you have a docstring in a class, it counts as "non-empty", so you don't need the "pass" to make it non-empty.